app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -76,19 +93,39 @@ export const AdditionalMarketData: React.FC<Props> = props => {
   return (
     <AdditionalMarketDataWrapper>
       <AdditionalMarketDataLeft>
-        <AdditionalMarketDataSectionWrapper href={`/#/24h-volume/category/${encodeURI(category)}`}>
+        <AdditionalMarketDataSectionWrapper href={`/#/24h-volume/category/${encodeURI(category)}`} noColorChange={true}>
           <IconCategory size={isMobile ? '20' : '24'} />
           <AdditionalMarketDataSectionTitle>{category}</AdditionalMarketDataSectionTitle>
         </AdditionalMarketDataSectionWrapper>
-        <AdditionalMarketDataSectionWrapper href={realitioUrl} rel="noopener noreferrer" target="_blank">
+        <AdditionalMarketDataSectionWrapper
+          data-arrow-color="transparent"
+          data-tip={`This market uses the ${oracle} oracle which crowed-sources the correct outcome.`}
+          href={realitioUrl}
+          rel="noopener noreferrer"
+          target="_blank"
+        >
           <IconOracle size={isMobile ? '20' : '24'} />
           <AdditionalMarketDataSectionTitle>{oracle}</AdditionalMarketDataSectionTitle>
         </AdditionalMarketDataSectionWrapper>
-        <AdditionalMarketDataSectionWrapper href={arbitrator.url} rel="noopener noreferrer" target="_blank">
+        <AdditionalMarketDataSectionWrapper
+          data-arrow-color="transparent"
+          data-tip={`This market uses the ${arbitrator.name} as the final arbitrator`}
```

Comment:
This reads better as 'This market uses ${arbitrator.name} as the final arbitrator'

Reply:
Should also have a period at the end of the sentence since we do that for the other tooltip

Reply:
will fix it! But in figma it was like that